ansible/roles/stackhpc.firewallgen/templates/sockets-to-rules.j2

```diff
@@ -0,0 +1,11 @@
+#jinja2: trim_blocks: "false", lstrip_blocks: "false"
```

Comment:
Sorting by (proto, port number, interface, destination) would help here. Looks like you can only sort by one attribute, so probably best to make it the port number.

Reply:
I went with (proto, interface, port number, destination) - I though tit was a bit easier to see what was going on with the interfaces grouped together.